nmigen/lib/cdc.py

```diff
@@ -125,13 +180,14 @@ class ResetSynchronizer(Elaboratable):
     Define the ``get_reset_sync`` platform method to override the implementation of
     :class:`ResetSynchronizer`, e.g. to instantiate library cells directly.
     """
-    def __init__(self, arst, *, domain="sync", stages=2, max_input_delay=None):
+    def __init__(self, arst, *, domain="sync", stages=2, max_input_delay=None, reset_less=False):
```

Comment:
This isn't used anymore, right?

Reply:
No it's not, it snuck in when I was cherry-picking. Oops...